test/lang/run_suite.py

```diff
@@ -4,7 +4,7 @@
 from sglang.test.test_utils import run_unittest_files
 
 suites = {
-    "minimal": ["test_srt_backend.py", "test_openai_backend.py"],
+    "minimal": ["test_srt_backend.py", "test_openai_backend.py", "test_srt_engine.py"],
```

Comment:
This engine is not related to the frontend, so please move this test under `test/srt/`

Reply:
done